### PR TITLE
Remove FFmpeg download dialog

### PR DESCRIPTION
**Pull request by phillip-hopper**
Remove the dialog and related code for downloading FFmpeg. Include FFmpeg in the installer now.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/saymore/90)

### CODE REVIEW

File: src/SayMore/UI/ConvertMediaDlg.cs [hunk 1]
```diff
@@ -157,7 +148,6 @@ private void SetupForAutoRun()
 		/// ------------------------------------------------------------------------------------
 		private void HandleStringsLocalized()
 		{
-			_labelDownloadNeeded.Tag = _labelDownloadNeeded.Text;
 			UpdateDisplay();
```
Comment (tombogle) [inline src/SayMore/UI/ConvertMediaDlg.cs]: This method is probably no longer needed. We'd only need to call UpdateDisplay when the localized strings change if we're doing something related to the strings (e.g., setting a variable or property to the new value).

File: src/SayMore/UI/ConvertMediaDlgViewModel.cs [hunk 2]
```diff
@@ -88,13 +89,6 @@ public string ConversionOutput
 			get { return _conversionOutput.ToString(); }
 		}
 
-		/// ------------------------------------------------------------------------------------
-		public void SetConversionStateBasedOnPresenceOfFfmpegForSayMore()
-		{
-			ConversionState = (FFmpegDownloadHelper.DoesFFmpegForSayMoreExist ?
```
Comment (tombogle) [inline src/SayMore/UI/ConvertMediaDlgViewModel.cs]: ConvertMediaUIState.FFmpegDownloadNeeded (and the code that uses it) can go away completely.

File: src/SayMore/UI/ConvertMediaDlg.cs (hunk 1 above)
Comment (phillip-hopper) [reply src/SayMore/UI/ConvertMediaDlg.cs]: Done.

File: src/SayMore/UI/ConvertMediaDlgViewModel.cs (hunk 2 above)
Comment (phillip-hopper) [reply src/SayMore/UI/ConvertMediaDlgViewModel.cs]: Done.

**Comment by phillip-hopper** [comment]
I removed the `license.txt` file from the `FFmpeg` directory. That file appears to be outdated (2007) compared to the website.

**Pull request merged**